xchange-okcoin/src/main/java/com/xeiam/xchange/okcoin/service/streaming/ChannelProvider.java

```diff
@@ -3,10 +3,17 @@
 import com.xeiam.xchange.currency.CurrencyPair;
 
 /** Returns market data channel name for given connection method and currency pair */
-interface ChannelProvider {
+public interface ChannelProvider {
```

Comment:
If you keep everything in com.xeiam.xchange.okcoin.service.streaming package you probabliy will not need to make this interface public which is good